Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:19.10
```

Comment:
What's the reason for not using an LTS ?

Reply:
To have access to newer packages.